tiny_dnn/layers/recurrent_cell_layer.h

```diff
@@ -28,6 +25,7 @@ namespace tiny_dnn {
  * x(t-1)    x(t)
  *
  **/
+// TODO (@prlz77) template might cause serialization errors. To be discussed.
```

Comment:
You shouldn't really use template parameter for layer classes.
If you do that, writing serialization routines will be tedious.